CHANGES/7917.bugfix

```diff
@@ -0,0 +1 @@
+Slow down sync when token is required
```

Comment:
Can we reword this to focus on what it's fixing instead? Perhaps `Avoid rate limiting by slowing down sync when token is required`.